### DISCUSSION TITLE
Please add Bluetooth Controller (Mouse/Keyboard)

### DISCUSSION THREAD

**Discussion by observator-x** [fdivitto/FabGL#162] [Ideas]
Please add Bluetooth Controller (Mouse/Keyboard) Driver, Thanks!

**Comment by UthdenTroll** [comment]
@fdivitto - has this been given any further consideration? I know this is from 2020, but thought I'd comment here rather than start a new subject. It's just that with limited space, and as someone who doesn't have PS/2 keyboards or mouse (although do have a VGA monitor) it would be great to have the option of a bluetooth or usb keyboard/mouse added.
